daml-assistant/daml-helper/src/DA/Daml/Helper/Main.hs

```diff
@@ -264,6 +265,9 @@ commandParser = subparser $ fold
             , command "allocate-parties" $ info
                 (ledgerAllocatePartiesCmd <**> helper)
                 (progDesc "Allocate parties on ledger")
+            , command "list-packages" $ info
```

Comment:
Have you considered `daml packages list`? That might make it easier to extend it with something like `daml packages get` to download an individual package.

Reply:
yes, happy to add the `packages` subcommand.